server/server/src/config.rs

```diff
@@ -75,11 +76,48 @@ impl ServerConfig {
     }
 }
 
+#[derive(Clone)]
+pub struct MetricsConfig {
+    pub duration_between_metrics_refresh: Duration,
+    pub duration_between_user_metrics: Duration,
+    pub duration_between_getting_pub_key_metrics: Duration,
+    pub duration_between_getting_pub_key_key_metrics: Duration,
+}
+
+impl MetricsConfig {
+    pub fn from_env_vars() -> MetricsConfig {
+        MetricsConfig {
+            duration_between_metrics_refresh: Duration::from_secs(
```

Comment:
seconds minutes?

Reply:
You can't do duration from_minutes. So when I retrieve the environment variable (which is in minutes), I just multiply it by 60 to get it in seconds.

Reply:
oh didn't see *60